atd-vze/src/views/Fatalities/Fatalities.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { withApollo } from "react-apollo";
+
+import { subYears } from "date-fns";
+import GridTable from "../../Components/GridTable";
+import gqlAbstract from "../../queries/gqlAbstract";
+import { fatalityGridTableColumns } from "./fatalityGridTableParameters";
+
+// Our initial query configuration
+let queryConf = {
+  table: "fatalities",
```

Comment:
Just thought about it a little more, but maybe we use `refetch` in some places to freshen up the data in a parent component like the crash and location tables. It would be hard to do that here since these two views are pretty disconnected.